example.py

```diff
@@ -13,6 +13,9 @@
 if client.logged_in: # check if client successfully logged in
     # get the all the periods (may return multiple types like trimesters and semesters but it doesn't really matter
     # the api will get it anyway)
+    nom_utilisateur = client.parametres_utilisateur['donneesSec']['donnees']['ressource']['L'] # get users name
```

Comment:
```suggestion
nom_utilisateur = client.info.name # get users name
```
You can actually access the name like this :)